### ISSUE TITLE
[Bug] OpenSCAD: Import produces extra "Lines" & one extra "Layer"

### ISSUE THREAD

**Issue by gneiss15** [FreeCAD/FreeCAD#7305]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Forums discussion

https://forum.freecadweb.org/viewtopic.php?f=10&t=70761

### Version

0.21 (Development)

### Full version info

```shell
FreeCAD_weekly-builds-29485-2022-07-24-conda-Linux-x86_64-py310.AppImage
[code]
OS: Ubuntu 18.04.6 LTS (ubuntu:GNOME/ubuntu)
Word size of FreeCAD: 64-bit
Version: 0.21.29485 (Git) AppImage
Build type: Release
Branch: master
Hash: a236ca843fdd6674afb6d7ed1454fbd3b547f5ea
Python 3.10.5, Qt 5.12.9, Coin 4.0.0, Vtk 9.1.0, OCC 7.5.3
Locale: German/Germany (de_DE)
Installed mods: 
  * Curves 0.5.2
  * fasteners 0.3.50
  * dxf-library
  * BIM 2021.12.0
  * fcgear 1.0.0
  * 3DfindIT 1.2.0
  * ArchTextures
  * sheetmetal 0.2.54
  * InventorLoader 1.3.0
  * Manipulator 1.4.9
  * ExplodedAssembly
  * Assembly4 0.12.3
  * 3D_Printing_Tools
  * nurbs
[/code]

```

### Subproject(s) affected?

OpenSCAD

### Issue description

As written in the title, the imported Object contains some extra parts (Lines & layer).
(There are other import-bugs, but I was requested to file a separate report for each)
Simply open the fle(s) in OpenScad and compare the 3D-View again the imported Object in FreeCad.

I had attached two scad-files.
The one that exhibits the bug is the file "CR2032.scad".
The other one is imported in correct oriantation.

[Scad-Files.zip](https://github.com/FreeCAD/FreeCAD/files/9253638/Scad-Files.zip)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
Is there a screenshot of the bug available ? can it be pasted to this ticket please ?

**Comment by chennes** [comment]
There are a few lines at the end of `importDXFface` that are commented out -- I'm not sure why. Those lines remove the extra draft lines that are being left in after importing text. A little bit of extra work needs to go into removing the final layer container.

**Comment by chennes** [comment]
I believe this is fixed by b3817cc3a67e48a0571270403bf35053279a9332 -- please re-open if that is not the case.
